### PR TITLE
[mcp23017] Added hint to the wiringPi library

### PR DESCRIPTION
**Pull request by cweitkamp**
Without this dependency the pi4j library did not work on my Raspberry Pi Model 3 B+.
Unfortunately there is another issue Pi4J/pi4j#319 which can be resolved by updating to pi4j release 1.2 (not yet released).

Supersedes #4806.

// CC @mrumpf 

Also-by: Michael Rumpf <michael@rumpfonline.de> (github: mrumpf)
Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

**Comment by cweitkamp** [comment]
@kaikreuzer Can you please add @aogorek to the team? Thanks.

File: addons/binding/org.openhab.binding.mcp23017/README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # MCP23017 Binding
 
 This binding allows you to have native access for MCP23017 I/O expander on I2C bus.
-It was tested with Raspberry PI 2 and Raspberry PI 3, but probably should work with other devices supported by pi4j library.
+It was tested with Raspberry PI 2 and Raspberry PI 3, but probably should work with other devices supported by [pi4j] (http://pi4j.com/) library.
```
Comment (wborn) [inline addons/binding/org.openhab.binding.mcp23017/README.md]:
```suggestion
It was tested with Raspberry Pi 2 and Raspberry Pi 3, but probably should work with other devices supported by [Pi4J](http://pi4j.com/) library.
```

File: addons/binding/org.openhab.binding.mcp23017/README.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 
-On Raspberry PI user on which openHAB is running (default user name is "openhab") needs to be added to groups "i2c" and  "gpio".
+On Raspberry PI the user on which openHAB is running (default user name is "openhab") needs to be added to groups "i2c" and  "gpio".
```
Comment (wborn) [inline addons/binding/org.openhab.binding.mcp23017/README.md]:
```suggestion
On Raspberry Pi the user on which openHAB is running (default user name is "openhab") needs to be added to groups "i2c" and  "gpio".
```

File: addons/binding/org.openhab.binding.mcp23017/README.md [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+
+## Dependencies
+
+Make sure that the [wiringPi](http://wiringpi.com/) library has been installed and that the `gpio` command line tool is available to openHAB.
+The shared library `libwiringPi.so` is required by the [pi4j](http://pi4j.com/) Java library to access the GPIO ports.
```
Comment (wborn) [inline addons/binding/org.openhab.binding.mcp23017/README.md]:
```suggestion
The shared library `libwiringPi.so` is required by the [Pi4J](http://pi4j.com/) Java library to access the GPIO ports.
```

**Comment by cweitkamp** [comment]
@wborn Why me? You did it :wink:!

**Comment by wborn** [comment]
> Unfortunately there is another issue [Pi4J/pi4j#319](https://github.com/Pi4J/pi4j/issues/319) which can be resolved by updating to pi4j release 1.2

Seems the official 1.2 release is being worked on: https://github.com/Pi4J/pi4j/issues/418#issuecomment-467432320

**Comment by kaikreuzer** [comment]
> @kaikreuzer Can you please add @aogorek to the team? Thanks.

Done.

**Comment by wborn** [comment]
Pi4J version 1.2 has been released @cweitkamp! Maybe you can update the library in a new PR. :-)

**Comment by cweitkamp** [comment]
Okay, I will do that.

**Comment by cweitkamp** [comment]
> Maybe you can update the library in a new PR.

Done.

**Pull request merged**